mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -1,17 +1,24 @@
 package ch.epfl.sdp.mobile.application
 
+import com.google.firebase.firestore.DocumentId
+
 /**
  * A document which represents the profile of a user. All the profile documents are stored in the
  * `/users/` collection.
  *
+ * @param uid the unique identifier for this profile.
  * @param name the human-readable name associated to this profile.
  * @param emoji the emoji associated with this profile.
  * @param backgroundColor the hex color code for this profile.
+ * @param uid the unique identifier for this profile.
+ * @param following a list of unique identifiers of the profiles this profile is following.
  */
 data class ProfileDocument(
+    @DocumentId val uid: String? = null,
     val name: String? = null,
     val emoji: String? = null,
-    val backgroundColor: String? = null
+    val backgroundColor: String? = null,
+    val followers: List<String>? = null
```

Comment:
I suggest adding a trailing comma.

```suggestion
    val followers: List<String>? = null,
```